lsp-ui-sideline.el

```diff
@@ -221,19 +222,19 @@ function signature)."
   (when contents
     (cond
      ((stringp contents) contents)
-     ((sequencep contents) ;; MarkedString[]
-      (seq-find (lambda (it) (and (hash-table-p it)
-                                  (lsp-get-renderer (gethash "language" it))))
+     ((vectorp contents) ;; MarkedString[]
+      (seq-find (lambda (it) (and (lsp-marked-string? it)
+                                  (lsp-get-renderer (lsp:marked-string-language it))))
                 contents))
-     ((gethash "kind" contents) contents) ;; MarkupContent
-     ((gethash "language" contents) contents))))
+     ((lsp:markup-content-kind contents) contents) ;; MarkupContent
```

Comment:
Shouldn't it better to check the type instead?
```suggestion
     ((lsp-markup-content? contents) contents) ;; MarkupContent
```

Reply:
we do it this way in lsp-mode